hook-docker/main.go

```diff
@@ -13,10 +13,11 @@ import (
 )
 
 type tinkConfig struct {
-	registry   string
-	baseURL    string
-	tinkerbell string
-	syslogHost string
+	registry      string
+	baseURL       string
+	tinkerbell    string
+	syslogHost    string
+	tinkServerTLS bool
```

Comment:
doesn't look like this value is being set anywhere. You might want to add to `parseCmdLine`.